source/NetFramework/Tests/TestCode/FSharp/EmptyType.fs

```diff
@@ -2,6 +2,10 @@ type Empty = class end
 
 (* il
 
+.assembly SharpLabAssembly

```

Comment:
Can we make it `_` instead of `SharpLabAssembly`? No good reason other than I always used `_` for assembly name in SharpLab, so it's consistent with errors messages and such.

Reply:
Changed to `_`.